src/scheduling/flow/cpu_cost_model.cc

```diff
@@ -898,7 +898,7 @@ FlowGraphNode* CpuCostModel::GatherStats(FlowGraphNode* accumulator,
       }
       // Running/idle task count
       rd_ptr->set_num_running_tasks_below(rd_ptr->current_running_tasks_size());
-      rd_ptr->set_num_slots_below(FLAGS_max_tasks_per_pu);
+      rd_ptr->set_num_slots_below(rd_ptr->max_pods());
```

Comment:
> 2)For PU node only, while updating the resource descriptor, update num_slots_below to max_pods like below.

@shivramsrivastava  We should update `set_num_slots_below` with  machine's `max_pods` here? (Get machine ResourceDescriptor using  `machine_res_id_pus_` and `machine_res_id` ?)